### DISCUSSION TITLE
Condemn being recommended when target is not on the health conditions

### DISCUSSION THREAD

**Discussion by gblnunes** [Hekili/hekili#1559] [Q&A] [answered]
Hello everyone,

I'm using Hekili for a few days and noticed that Condemn, for me, is recommended when the target is not on the health threshold (above 80% or below 20%) so Condemn stay on the recommended list but can't be used.

I read the Issues and didn't find anything talking about this so I think I'm doing something wrong.

Can someone help?

**Comment by Hekili** [comment, marked as answer]
Please follow the ticket submission instructions and submit as an Issue with all the required information.  This would be a "Recommendations" ticket.

https://github.com/Hekili/hekili/issues

More likely than not, you're seeing this in multi-target scenarios where another target could be Condemned, but I can't tell without the proper information submitted.  Thanks!
